### PR TITLE
add branch preview doc to dev docs - FOLIO-2022

### PR DESCRIPTION
**Pull request by funkymalc**
@dcrossleyau - Can you take a quick look and let me know if any changes are required?

### CODE REVIEW

File: guides/branch-preview.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+naming convention is 'GITHUB_PROJECT NAME_ + PR_NUMBER_ + JENKINS_BUILD_NUMBER'.  
+PR 64 for ui-users would create a tenant called 'ui_users_64_1', for example.  The tenant
+admin user for the tenant is the same as the tenant name + 'admin'.  For example, the tenant
+admin user for tenant, ui_users_64_1, would be 'ui_users_64_1_admin'.  The tenant admin password
+in all cases is 'admin'.  
+
+* Tenant modules are enabled and reference and sample data for select modules are loaded 
+for the tenant. 
+
+* The stripes webpack is distributed to an S3 web service.  An AWS S3 URL to the UI is provided.
```
Comment (dltj) [inline guides/branch-preview.md]: Whoa!  This is nice.  One clarification, for _"The stripes webpack is distributed to an S3 web service. An AWS S3 URL to the UI is provided."_, how/where is the URL to the UI provided?

Comment (dcrossleyau) [reply guides/branch-preview.md]: Jenkins appends it to the PR. Enhanced the doc for that. Thanks.

**Comment by dltj** [comment]
Do you mean that Jenkins adds it as a comment to the pull request?

**Comment by dcrossleyau** [comment]
Yes.

**Comment by dcrossleyau** [comment]
Please say if you reckon that the published document is still not clear enough and i will amend.

https://dev.folio.org/guides/branch-preview/

**Pull request merged**